framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Form/AbpDynamicformTagHelper.cs

```diff
@@ -12,6 +12,9 @@ public class AbpDynamicFormTagHelper : AbpTagHelper<AbpDynamicFormTagHelper, Abp
     [HtmlAttributeName("abp-model")]
     public ModelExpression Model { get; set; }
 
+    [HtmlAttributeName("abp-column")]
+    public int Column { get; set; }
```

Comment:
I think we can use `ColumnSize` enum here, instead of int.

https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Grid/ColumnSize.cs